aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1059,65 +1058,6 @@ impl AptosVM {
         ))
     }
 
-    fn simulate_multisig_transaction<'a, 'r>(
```

Comment:
So, a multisig transaction can't be simulated anymore? If I recall correctly, a wallet first simulates a transaction and displays gas estimates, etc?

Reply:
`execute_multisig_transaction` doesn't seem to take any `is_simulation: bool` flag as input? Is it because the logic of running an actual execution vs simulation is similar?

Reply:
Yes, it was made the same with `is_transaction_simulation_enhancement_enabled` feature flag. Basically we had `execute_or_simulate_multisig_transaction` which called `simulate_multisig_transaction` if simulation with flag off, and `execute_multisig_transaction` if not simulation, or simulation with flag on. The flag is on now, so that part of code is unreachable for the nodes